packages/plugin-odoo/src/NoteEditableXmlDomParser.ts

```diff
@@ -0,0 +1,30 @@
+import { XmlDomParsingEngine } from '../../plugin-xml/src/XmlDomParsingEngine';
+import { DividerNode } from '../../plugin-divider/src/DividerNode';
+import { nodeName } from '../../utils/src/utils';
+import { DividerXmlDomParser } from '../../plugin-divider/src/DividerXmlDomParser';
+import { AtomicNode } from '../../core/src/VNodes/AtomicNode';
+
+export class NoteEditableXmlDomParser extends DividerXmlDomParser {
+    static id = XmlDomParsingEngine.id;
+    engine: XmlDomParsingEngine;
+
+    predicate = (item: Node): boolean => {
+        return (
+            item instanceof Element &&
+            nodeName(item) === 'DIV' &&
+            item.classList.contains('note-editable')
+        );
+    };
+
+    async parse(item: Element): Promise<DividerNode[]> {
+        const divider = (await super.parse(item))[0];
+
+        const children = divider.childVNodes;
+        if (children.every(child => child instanceof AtomicNode)) {
+            const container = new this.engine.editor.configuration.defaults.Container();
+            container.append(...children);
+            divider.append(container);
```

Comment:
Wait so... every atomic child of the `note-editable` div is put into its own personal container ? Wouldn't it be more logical to have them all in the same container ? Besides, wouldn't your bug still exist if only some of the nodes are atomic but not all of them ? And wouldn't thix fix create additional `div` after rendering ? This fix here basically looks extremely fishy to me.

Reply:
You're right, I forgot to consider the case where there is some node that are not Atomic. I'm adapting the code.
This fix indeed create additional `div` after the rendering. 

That's a trade-off we were discussing with @Zinston. The more appropriate fix is to handle the `InlineNode` inside a container but that would require more work.

Reply:
What is the actual bug (steps to reproduce) this is trying to solve ?

Reply:
Try to edit the signature in Odoo. The content is 
```html
<span data-o-mail-quote="1">--&nbsp;</span>
<br data-o-mail-quote="1">
<span data-o-mail-quote="1">Administrator</span>
```

Reply:
I changed the code to take into consideration nodes that are not inline.